src/redux/configureStore.js

```diff
@@ -0,0 +1,17 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import { loadingBarReducer } from 'react-redux-loading-bar';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import countriesReducer from './countries/countries';
+
+const reducer = combineReducers({
+  countries: countriesReducer,
+  loadingBar: loadingBarReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(thunk, logger),
+);
```

Comment:
- [ ] Do not log redux data on the browser console as it may cause site security issues. Kindly clear the logged data and the errors displayed on the browser console.